src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -382,6 +384,26 @@ public void runInPodWithLivenessProbe() throws Exception {
         r.assertLogContains("Still alive", b);
     }
 
+    @Test
+    public void podTemplateWithMultipleLabels() throws Exception {
+        PodTemplate pt = new PodTemplate();
+        pt.setName("podTemplate");
+        pt.setLabel("label1 label2");
+        ContainerTemplate jnlp = new ContainerTemplate("jnlp", "jenkins/jnlp-slave:3.35-5-alpine");
+        pt.setContainers(Collections.singletonList(jnlp));
+        cloud.addTemplate(pt);
+        SemaphoreStep.waitForStart("pod/1", b);
+        Map<String, String> labels = getLabels(cloud, this, name);
```

Comment:
Do we also expect that the `Computer` has the original labels `label1` & `label2`? I guess https://github.com/jenkinsci/kubernetes-plugin/blob/be4b55ffc3f85eca842fbb42cdaa7f8937a8747d/src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/podTemplateWithMultipleLabels.groovy#L1 tests this implicitly but for the benefit of readers it might be good to assert this explicitly.